tendrl/gluster_integration/manager/manager.py

```diff
@@ -69,16 +73,20 @@ def _run(self):
 class GlusterIntegrationManager(Manager):
     def __init__(self, cluster_id):
         self._complete = gevent.event.Event()
+        etcd_kwargs = {
+            'port': config['etcd_port'],
```

Comment:
We probably need to discuss and close out on whether we are going with the approach here or config['configuration']['etcd_port'] as in https://github.com/Tendrl/node-agent/pull/143/files as I am also following that approach in my monitoring PR s.